app/components/containers/containersController.js

```diff
@@ -128,7 +130,11 @@ angular.module('containers', [])
           });
         }
         else if (action === Container.remove) {
-          action({id: c.Id}, function (d) {
+          var force = false;
+          if (c.State === 'running') {
```

Comment:
If we always force, we have to take care if other functions call directly `removeAction`, cause it's now a forceRemoveAction